public/js/all-breaches/all-breaches.js

```diff
@@ -132,6 +136,7 @@ function initBreaches() {
     const [fuzzyShowAll, showHiddenBreaches] = document.querySelectorAll(".show-all-breaches");
 
     showHiddenBreaches.addEventListener("click", (e) => {
+      sendPing(e.target, "Click", "All Breaches Page");
```

Comment:
Could we achieve the same effect (i.e., to send a GA ping) by adding `send-ga-ping` class and data attributes?